static/main.css

```diff
@@ -272,6 +272,18 @@ a {
   color: #ffffff;
 }
 
+ .orange-background-button:disabled {
+  width: auto;
+  height: auto;
+  padding: 9px 25px 7px 24px;
+  border-radius: 24px;
+  border: solid 2px #dadde1;
+  background-color: #ffffff;
+  font-weight: bold;
+  font-size: 14px;
+  color: #dadde1;
+}
+
 .notification-button, .notification-button:hover, .notification-button:active {
```

Comment:
Other button types are missing the disabled option. The disabled attribute can be added to a higher hierarchy to apply to all buttons. Thanks.

Reply:
my local code has the shadow css. however, i already commited all the code to github. there is nothing i can commit the code. But it seams the code on github doesnot have the shadow css.